docker/airflow/bootstrap

```diff
@@ -28,6 +28,9 @@ if [[ -n ${KMS_DATA_KEY} ]]; then
 		echo "Decrypting POSTGRES_PASSWORD_ENC..."
 		export POSTGRES_PASSWORD=$(openssl enc -d -aes256 -k <(echo "${kms_data_key_plaintext}") \
 			-in <(echo "${POSTGRES_PASSWORD_ENC}" | base64 -d))
+	elif [[ -n "${SSM_POSTGRES_PASSWORD}" ]]; then
+		echo "Fetching SSM_POSTGRES_PASSWORD..."
+		export POSTGRES_PASSWORD=$(aws ssm get-parameter --name "${SSM_POSTGRES_PASSWORD}" --with-decryption --query "Parameter.Value" --output text)
```

Comment:
- What does `SSM_POSTGRES_PASSWORD` mean. Is it the password, or the SSM key path?
- `SSM_POSTGRES_PASSWORD` doesn't depend on `KMS_DATA_KEY`. If `KMS_DATA_KEY` is provided, I would assume all `*_ENC` fields need to be decrypted.
- You could check for `KMS_DATA_KEY` and then decrypt all `*_ENC` environment variables and if its not provided, get corresponding stuff from SSM. Because its a public repo, it'll be much appreciated.

Reply:
Anything that is prefixed with SSM is going to be the path, otherwise SSM doesn't mean anything. 

I didn't add a completely new switch because I don't have a structure worked out for the other keys yet, but I can wait to merge it until I'm ready for that. This is also the reason why SSM_POSTGRES_PASSWORD is ignored if POSTGRES_PASSWORD_ENC is present.